### ISSUE TITLE
Insufficient color contrast for primary menu chevron icons

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.org#305]
Images that aren’t decorative like those icons need a color contrast of at least 3:1. Currently `primary-nav__icon` elements use #c8c8c8 on white.

**Comment by thibaudcolas** [comment]
Fixed in #314.
